jbehave-support-core/src/test/groovy/org/jbehavesupport/core/TestConfig.groovy

```diff
@@ -98,14 +99,14 @@ class TestConfig {
             .logPath(env.getProperty("ssh.logPath"))
             .build()
 
-        RollingLogResolver rollingLogResolver = new SimpleRollingLogResolver();
+        RollingLogResolver rollingLogResolver = new SimpleRollingLogResolver()
         return new SshTemplate(passwordSetting, env.getProperty("ssh.timestampFormat"), rollingLogResolver)
     }
 
     @Bean
     @Qualifier("TEST")
     WebServiceHandler requestFactoryTestHandler() {
-        return new WebServiceHandler() {
+        return new TestWebServiceHandler(env) {
```

Comment:
Just a question - why was this change needed?

Reply:
The `TestWebServiceHandler` is set up to acces the mock app without the need for further configuration